tests/tests/test_stdcm.py

```diff
@@ -266,7 +266,75 @@ def test_max_running_time(small_scenario: Scenario, fast_rolling_stock: int):
     r = requests.post(url, json=payload)
     response = r.json()
     assert r.status_code == 200
-    assert response == {"status": "path_not_found"}
+    assert response == {
+        "status": "conflicts",
+        "conflicts": [
+            {
+                "conflict_type": "Spacing",
+                "start_time": "2024-01-01T07:30:00Z",
+                "end_time": "2024-01-01T16:00:00Z",
```

Comment:
If the train starts at 7:30, the conflict start time should be later than this (it happens on the last track). And the end time should be earlier, the path isn't that long.

(same comment for requirement times)

Reply:
Resolving this, this weird behavior comes from core. I'll double check what's the intended behavior and open a different issue if relevant.